net/openconnect/files/openconnect.sh

```diff
@@ -76,11 +77,23 @@ proto_openconnect_setup() {
 	logger -t openconnect "initializing..."
 
 	[ -n "$interface" ] && {
+		local trials=10
+
 		logger -t "openconnect" "adding host dependency for $server at $config"
-		for ip in $(resolveip -t 10 "$server"); do
-			logger -t "openconnect" "adding host dependency for $ip at $config"
-			proto_add_host_dependency "$config" "$ip" "$interface"
+		resolveip -t "$server" >"$tmpfile"
```

Comment:
Checking sh status variable "$?" directly is sort of non-standard, I'd be inclined to write this something like this (which also gets rid of the double resolveip calls.  (Also, although it works in busybox ash, I'm not sure I've seen "let" used in any of the core sh scripts, I see everyone using the POSIX arithmetic.)
```
while resolveip -t "$server" >"$tmpfile" && [ "$trials" -gt 0 ]; do
        sleep 5
        trials=$((trials - 1))
done
```

Reply:
Thanks